internal/testing/e2e/consul.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	defaultConsulImage                = "hashicorp/consul:1.11.4"
+	defaultConsulImage                = "hashicorp/consul-enterprise:1.11-ent"
```

Comment:
This means that every time we run e2e tests we need a Consul Enterprise license, right? Is this an intentional change?

Reply:
Nope that was just for testing on my local. I will say in the CI matrix that value gets overwritten by the env.